source/components/description-list/index.html

```diff
@@ -0,0 +1,109 @@
+---
+title: Description List
+assets:
+  - helix-ui.css
+---
+{% extends 'component.njk' %}
+{% block content %}
+
+<section>
+  <h2 class="hxSectionTitle" id="default-list">Default List</h2>
+  <p>
+    Use a Description List <code>hx-dl</code> along with children Definition
+    <code>hx-def</code>, Term <code>hx-dt</code>, and Description Detail
+    <code>hx-dd</code> custom element tags. These are all required to construct
+    a Description List.
+  </p>
+  <div class="demo">
+    <hx-dl>
+      <hx-def>
+        <hx-dt>First Key Term</hx-dt>
+        <hx-dd>This is the description and/or data value.</hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Second Key Term</hx-dt>
+        <hx-dd>
+          This is the description and/or data value. This paragraph shows the
+          default wrapping that exists on this element. Description Detail tags
+          will wrap across multiple lines by default.
+        </hx-dd>
+      </hx-def>
+      <hx-def>
+        <hx-dt>Third Key Term</hx-dt>
+        <hx-dd class="hxForceWordBreak">
+          This is the description and/or data value. This third
+          description is an example of when you may have long strings
+          (https://swapi.co/api/people/1/) in your content that usually
```

Comment:
should the word `content` be wrapping? currently it is wrapping to the next line / https://randomkeygen.com/ is a site where you can grab a random key from :) (long ones are towards the bottom)

Reply:
I'll second this. You can update the key to be something like "API Key" along with `hx-dd.hxForceWordBreak` containing a really long (fake) API key. 

At minimum,  documentation of the helper class should be moved to a paragraph above the demo.